### ISSUE TITLE
No error log when uploading a malformed helm chart to Harbor.

### ISSUE THREAD

**Issue by reasonerjt** [vmware/harbor#5473]
Build: Version v1.6.0-c5a79fb5

Go to Helm Charts tab click upload and select a .yaml file as a helm chart, there's an error as expected.
![image](https://user-images.githubusercontent.com/2390463/43568127-661df55e-9666-11e8-93cf-0720de2d81d0.png)
But when I check the log in ui.log,  I don't see any error.

There should be explicit error at least in the log file telling user the chart is malformed.

**Comment by steven-zou** [comment]
There should be logical formatted error message returned by the API:
`{"error": "error message"}`

**Comment by steven-zou** [comment]
```
{error: "chart metadata (Chart.yaml) missing"}
"chart metadata (Chart.yaml) missing"
```

**Comment by steven-zou** [comment]
assigned to @ninjadq to add more detailed error message in frontend layer

**Comment by zhoumeina** [comment]
fixed by pr #5537
